src/app/RoutingRules.php

```diff
@@ -75,6 +76,7 @@ public static function installRoutes($app)
     $app->get('/market/{key}[/{version}]', ProductAction::class);
     $app->get('/_market/{key}/_meta.json', MetaJsonAction::class);
     $app->get('/_market/{key}/openapi', OpenApiJsonAction::class);
+    $app->get('/internal/market-rcptt', MarketRCPTTAction::class);
```

Comment:
don't know if I fully got it: you just fake the market version to be tested?  your teams client is off ;)

Reply:
how far will 'generate rcptt on the fly' go? Where do we define exclusions, e.g. whitelist expected warnings or the like?